### PR TITLE
Feat: "added Navbar"

### PR DESCRIPTION
**Pull request by sadathshariff**
Added a simple navbar
navbar

### CODE REVIEW

File: CSS/Components/navbar.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.nav-container {
+  width: 100%;
+}
+.navbar-container {
+  background-color: var(--primary-color);
+  color: var(--off-white);
+  display: flex;
+  justify-content: space-between;
```
Comment (tanishq20) [inline CSS/Components/navbar.css]: `justify-content: space-between` may cause problems with logo responsive as told by Vikram sir.
Try to do it with `margin-left: auto`

**Review by tanishq20 — COMMENTED** [review]
Suggested small change.

File: CSS/Components/navbar.css (hunk 1 above)
Comment (sadathshariff) [reply CSS/Components/navbar.css]: Thanks Tanishq,Made the changes

**Review by vivekmehto — APPROVED** [review]
Looks good.

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**